hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/state/HdfsStore.java

```diff
@@ -0,0 +1,21 @@
+package org.apache.mesos.hdfs.state;
```

Comment:
We shouldn't need an interface for this class. Once we remove getFrameworkId and setFrameworkId it will be more of a utility class.